src/bun.js/bindings/ZigGlobalObject.cpp

```diff
@@ -501,6 +506,13 @@ extern "C" JSC::JSGlobalObject* Zig__GlobalObject__create(void* console_client,
     Bun__setDefaultGlobalObject(globalObject);
     JSC::gcProtect(globalObject);
 
+#ifdef FUZZILLI_ENABLED
+    // Register fuzzilli() function if in fuzzilli mode
+    if (std::getenv("BUN_FUZZILLI_MODE")) {
```

Comment:
can be cleaned up later, but we can probably remove the BUN_FUZZILLI_MODE and only use FUZZILLI_ENABLED